### ISSUE TITLE
Cannot build: “Failed to unpack package `expat-sys v2.1.1`”

### ISSUE THREAD

**Issue by ProgVal** [servo/servo#9466]
```
$ ./mach build -r -v      
looking for rustc at /home/progval/src/servo/.servo/rust/2015-12-27/rustc-nightly-x86_64-unknown-linux-gnu/rustc/bin/rustc
Downloading Rust compiler...
Downloading Rust compiler: 100.0%
Extracting Rust compiler...
Downloading Host rust library for target x86_64-unknown-linux-gnu...
Downloading Host rust library for target x86_64-unknown-linux-gnu: 100.0%
Extracting Rust stdlib for target x86_64-unknown-linux-gnu...
Downloading Host rust library for target arm-linux-androideabi...
Downloading Host rust library for target arm-linux-androideabi: 100.0%
Extracting Rust stdlib for target arm-linux-androideabi...
Rust ready.
Downloading Cargo nightly...
Downloading Cargo nightly: 100.0%
Extracting Cargo nightly...
Cargo ready.
cargo build --release -v
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/servo/rust-layers`
    Updating git repository `https://github.com/servo/ipc-channel`
    Updating git repository `https://github.com/ecoal95/rust-offscreen-rendering-context`
    Updating git repository `https://github.com/pcwalton/gaol`
    Updating git repository `https://github.com/servo/rust-azure`
    Updating git repository `https://github.com/servo/rust-freetype`
    Updating git repository `https://github.com/TyOverby/bincode`
    Updating git repository `https://github.com/Manishearth/rust-tenacious`
    Updating git repository `https://github.com/huonw/simd`
    Updating git repository `https://github.com/aweinstock314/rust-clipboard`
    Updating git repository `https://github.com/libheartbeats/heartbeats-simple-rust.git`
    Updating git repository `https://github.com/libheartbeats/heartbeats-simple-sys.git`
    Updating git repository `https://github.com/Ygg01/xml5ever`
    Updating git repository `https://github.com/ecoal95/angle`
    Updating git repository `https://github.com/servo/rust-mozjs`
    Updating git repository `https://github.com/servo/mozjs`
    Updating git repository `https://github.com/ende76/brotli-rs`
    Updating git repository `https://github.com/energymon/energymon-rust.git`
    Updating git repository `https://github.com/energymon/energymon-sys.git`
    Updating git repository `https://github.com/jgraham/webdriver-rust.git`
 Downloading servo-egl v0.2.0
 Downloading heapsize_plugin v0.1.2
 Downloading wayland-sys v0.5.8
 Downloading websocket v0.14.0
 Downloading lazy_static v0.1.15
 Downloading user32-sys v0.1.2
 Downloading malloc_buf v0.0.6
 Downloading objc v0.1.8
 Downloading phf v0.7.5
 Downloading tendril v0.2.1
 Downloading gcc v0.3.17
 Downloading mime_guess v1.3.0
 Downloading dwmapi-sys v0.1.0
 Downloading traitobject v0.0.1
 Downloading ws2_32-sys v0.2.0
 Downloading io-surface v0.1.2
 Downloading crossbeam v0.1.6
 Downloading encoding-index-simpchinese v1.20141219.5
 Downloading libc v0.2.4
 Downloading libressl-pnacl-sys v2.1.6
 Downloading rustc-serialize v0.3.16
 Downloading servo-fontconfig-sys v2.11.2-really.1
 Downloading gleam v0.2.2
 Downloading winapi v0.2.5
 Downloading encoding v0.2.32
 Downloading unreachable v0.0.2
 Downloading android_glue v0.1.3
 Downloading env_logger v0.3.1
 Downloading utf-8 v0.6.0
 Downloading color_quant v1.0.0
 Downloading heapsize v0.2.0
 Downloading wayland-window v0.2.2
 Downloading image v0.5.4
 Downloading aho-corasick v0.4.0
 Downloading quasi v0.3.10
 Downloading encoding_index_tests v0.1.4
 Downloading unicode-bidi v0.2.2
 Downloading wayland-kbd v0.3.4
 Downloading core-text v1.0.0
 Downloading fnv v1.0.0
 Downloading memmap v0.2.3
 Downloading serde v0.6.6
 Downloading language-tags v0.2.0
 Downloading servo-freetype-sys v2.4.11
 Downloading net2 v0.2.19
 Downloading caseless v0.1.1
 Downloading core-foundation v0.2.0
 Downloading void v0.0.5
 Downloading regex v0.1.43
 Downloading quasi_macros v0.3.9
 Downloading phf_codegen v0.7.5
 Downloading unicode-normalization v0.1.1
 Downloading log v0.3.4
 Downloading wayland-scanner v0.5.8
 Downloading winapi-build v0.1.1
 Downloading inflate v0.1.0
 Downloading openssl-sys-extras v0.7.1
 Downloading url v0.5.2
 Downloading debug-builders v0.1.0
 Downloading openssl-sys v0.7.1
 Downloading rand v0.3.12
 Downloading stb_image v0.2.1
 Downloading advapi32-sys v0.1.2
 Downloading make-cmd v0.1.0
 Downloading pnacl-build-helper v1.4.10
 Downloading quickersort v1.0.0
 Downloading memchr v0.1.7
 Downloading encoding-index-japanese v1.20141219.5
 Downloading ref_slice v0.1.0
 Downloading encoding-index-tradchinese v1.20141219.5
 Downloading cocoa v0.2.2
 Downloading regex-syntax v0.2.2
 Downloading debug_unreachable v0.0.6
 Downloading gdi32-sys v0.1.1
 Downloading euclid v0.4.0
 Downloading num_cpus v0.2.10
 Downloading selectors v0.2.3
 Downloading block v0.1.4
 Downloading typeable v0.1.2
 Downloading phf_generator v0.7.5
 Downloading khronos_api v1.0.0
 Downloading encoding-index-singlebyte v1.20141219.5
 Downloading libz-sys v1.0.0
 Downloading osmesa-sys v0.0.5
 Downloading num v0.1.28
 Downloading aster v0.9.2
 Downloading unicase v1.0.1
 Downloading encoding-index-korean v1.20141219.5
 Downloading phf_shared v0.7.5
 Downloading tempdir v0.3.4
 Downloading unicode-script v0.1.1
 Downloading x11-dl v2.3.0
 Downloading servo-glutin v0.4.8
 Downloading flate2 v0.2.11
 Downloading kernel32-sys v0.2.1
 Downloading httparse v1.1.0
 Downloading time v0.1.34
 Downloading string_cache v0.2.5
 Downloading libc v0.1.12
 Downloading fs2 v0.2.2
 Downloading dylib v0.0.1
 Downloading cookie v0.2.2
 Downloading tempfile v1.1.3
 Downloading getopts v0.2.14
 Downloading quasi_codegen v0.3.10
 Downloading objc-foundation v0.0.2
 Downloading xml-rs v0.2.2
 Downloading serde_codegen v0.6.6
 Downloading servo-fontconfig v0.2.0
 Downloading gif v0.6.0
 Downloading hpack v0.2.0
 Downloading dlib v0.2.0
 Downloading expat-sys v2.1.1
unable to get packages from source

Caused by:
  Failed to unpack package `expat-sys v2.1.1`

Caused by:
  No such file or directory (os error 2)
Build completed in 213.51s
```

OS: Debian 8.3 (Jessie)

**Comment by nox** [comment]
https://github.com/servo/servo/pull/9467

Sorry for the inconvenience, should be fixed ASAP.

**Comment by Wafflespeanut** [comment]
should've been fixed by #9467

**Comment by sunose** [comment]
today build servo under windows 10 msys2 -64,this error happend again.

please help check this.
